src/EnergyPlus/RuntimeLanguageProcessor.cc

```diff
@@ -2240,9 +2240,9 @@ namespace RuntimeLanguageProcessor {
 				//                        PsyTsatFnPb(Operand(1)%Number, & ! pressure (Pa)
 				//                                    'EMS Built-In Function') )
 			} else if ( SELECT_CASE_var == FuncCpCW ) {
-				ReturnValue = SetErlValueNumber( CPCW( Operand( 1 ).Number ) ); // result => specific heat of water (J/kg-K) = 4180.d0 | temperature (C) unused
+				ReturnValue = SetErlValueNumber( CPCW() ); // result => specific heat of water (J/kg-K) = 4180.d0 | temperature (C) unused
 			} else if ( SELECT_CASE_var == FuncCpHW ) {
```

Comment:
@Myoldmopar Noticing these Erl functions - they'll have to remain even though they'll simply return the constant 'cause we don't want to get into trying to transition erl code (yikes!).  But an I/O ref update to note that these simply return value x and y would be in order.